src/check_result.jl

```diff
@@ -137,3 +143,12 @@ function _check_add!!_behaviour(acc, val; kwargs...)
     acc_mutated = deepcopy(acc)  # prevent this test changing others
     check_equal(add!!(acc_mutated, val), acc + val; kwargs...)
 end
+
+# `+` is not defined for `NotImplemented`
```

Comment:
it is now, right?

Reply:
Yes, I had noticed this as well, it is fixed now.